hack/tools/tilt-prepare/main.go

```diff
@@ -604,12 +602,7 @@ func cleanupChartTask(path string) taskFunction {
 
 				helmChartVersion := helmChart.Version
 				if helmChartVersion == "" {
-					// If helmChart.Version is not set lookup the latest version.
-					helmChartVersion, err = resolveLatestHelmChartVersion(ctx, helmChart.Repo, helmChart.Name)
-					if err != nil {
-						return err
-					}
-					klog.Infof("[%s] resolved latest Helm Chart version to %q\n", prefix, helmChartVersion)
+					return errors.New("Helm Chart version must be set")
```

Comment:
I had to get rid of the helm dependency as it's not compatible with the new docker/docker version (because they both depend on incompatible versions of oras.land/oras-go

We previously used this code to pull latest Helm Chart versions. But we stopped using that to have a stable dev environment.

I think it's fine to drop this code as we don't want to go back to a dev-env that can break any day when new Helm Charts are released.

Reply:
@Jont828 fyi

Reply:
@sbueringer So just to clarify, would that break using the visualizer app with Tilt, or would we still be good to use it as we are now?

Reply:
This does not break the visualizer app.

The consequence is just that we always have to set a helm chart version in our kustomization.yamls. But we already do this everywhere for our own sanity, e.g. https://github.com/kubernetes-sigs/cluster-api/blob/e5db021e59d38f916684c2bd8f374dde7805cf9e/hack/observability/visualizer/kustomization.yaml#L10

Before this PR we could have left the version unset and tilt-prepare would have automatically pulled latest

Reply:
CAPZ has deeper integration with Helm, and it's not possible to remove those imports.

@sbueringer do you have any ideas about how to work around this `no export data for "oras.land/oras-go/pkg/auth/docker"` problem caused by importing both Docker and Helm? I've been trying a variety of approaches but no luck so far, and it's preventing CAPZ from adopting CAPI v1.4.5 and v1.5.0.